isaric/parsers/isaric-rapid.toml

```diff
@@ -593,10 +593,29 @@
 [visit.treatment_antimalarial]
   description = "Antimalarial"
   combinedType = "any"
+  excludeWhen = "none"
   fields = [
     { field = "overall_antimal_cmyn", ref = "Y/N/NK", description = "Antimalarial agent?" },
     { field = "daily_antimal_cmyn", ref = "Y/N/NK", description = "Antimalarial agent?" },
     { field = "antimal_cmyn", ref = "Y/N/NK", description = "Antimalarial agent on the first day of admission?" },
+    { field = "overall_exper_cmtype___1", values = { 1 = true } },
+    { field = "overall_exper_cmtype___2", values = { 1 = true } },
+    { field = "daily_exper_cmtype___1", values = { 1 = true } },
+    { field = "daily_exper_cmtype___2", values = { 1 = true } },
+    { field = "exper_cmtype___1", values = { 1 = true } },
+    { field = "exper_cmtype___2", values = { 1 = true } },
+  ]
+
+[visit.treatment_antimalarial_type]
+  combinedType = "set"
```

Comment:
@ekamau I don't think other antimalarial treatments as indicated in antimal_cmtype, daily_antimal_cmtype and overall_antimal_cmtype have all been included - I couldn't see these in the parser in the main branch and the edits here only include chloroquine and hydroxychloroquine. So these additional ones could be mapped also (primaquine, quinine/mefloquine etc.)

Reply:
Thanks @sadiekelly  - I have added the extra variables